testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddTrustManagerOfflineTest.java

```diff
@@ -0,0 +1,254 @@
+package org.wildfly.extras.creaper.commands.elytron.tls;
+
+import static org.junit.Assert.fail;
+import static org.wildfly.extras.creaper.XmlAssert.assertXmlIdentical;
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import java.io.File;
+import org.custommonkey.xmlunit.XMLUnit;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.ManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineManagementClient;
+import org.wildfly.extras.creaper.core.offline.OfflineOptions;
+
+public class AddTrustManagerOfflineTest {
```

Comment:
Key stores and key managers used on a couple of places (constants, method names, fail descriptions), I would prefer usage of trust stores and trust managers